test/test_container.py

```diff
@@ -602,7 +597,7 @@ class Service(object):
 
             @dummy
             def method(self):
-                pass
+                pass  # pragma: no cover
```

Comment:
The entrypoints need to be defined for various reasons (e.g. here the class would not be considered a service without one) but they're never actually executed.